web/src/features/modals/SettingsModal.tsx

```diff
@@ -1,108 +1,330 @@
+      <span className="text-sm font-medium text-secondary dark:text-secondary-dark">
+        {t('legends.colorblindmode')}
+      </span>
+      <SwitchToggle
+        isEnabled={isEnabled}
+        onChange={onToggle}
+        ariaLabel={t('legends.colorblindmode')}
+      />
+    </div>
+  );
+}
+
+function LanguageSelectorToggle() {
+  const { t, i18n } = useTranslation();
+  const currentLanguageKey = i18n.languages[0] as keyof typeof languageNames;
+  const selectedLanguage = languageNames[currentLanguageKey] ?? 'English';
+
+  const [isOpen, setIsOpen] = useState(false);
+  const buttonReference = useRef<HTMLButtonElement>(null);
+
+  const handleToggle = useCallback(() => setIsOpen(!isOpen), [isOpen]);
+  const handleClose = useCallback(() => setIsOpen(false), []);
+  const handleKeyDown = useCallback(
+    (event: React.KeyboardEvent) => {
+      if (event.key === 'Enter' || event.key === ' ') {
+        handleToggle();
+      }
+    },
+    [handleToggle]
+  );
+
+  return (
+    <div className="w-full p-2">
+      <button
+        ref={buttonReference}
+        className="flex w-full items-center justify-between"
+        onClick={handleToggle}
+        onKeyDown={handleKeyDown}
+      >
+        <span className="text-sm font-medium text-secondary dark:text-secondary-dark">
+          {t('Language')}
+        </span>
+        <span className="text-sm text-secondary dark:text-secondary-dark">
+          {selectedLanguage}
+        </span>
+      </button>
+      {isOpen && <LanguageSelector parentRef={buttonReference} onClose={handleClose} />}
+    </div>
+  );
+}
+
+function AboutElectricityMaps() {
+  const { t } = useTranslation();
+  const [isCollapsed, setIsCollapsed] = useState(true);
+
+  return (
+    <div className="w-full px-2 pt-2">
+      <Accordion
+        title={
+          <p className="text-secondary dark:text-secondary-dark">{t('info.title')}</p>
         }
+        isCollapsed={isCollapsed}
+        setState={setIsCollapsed}
       >
-        {t(
-          isEnabled
-            ? `tooltips.hide${typeAsTitlecase}Layer`
-            : `tooltips.show${typeAsTitlecase}Layer`
-        )}
-      </Button>
-    </>
+        <div className="mt-2 text-xs text-secondary dark:text-secondary-dark">
+          <p className="mb-4">
+            <Link href="https://electricitymaps.com">Electricity Maps</Link> offers an API
+            that delivers real-time and predictive electricity grid signals.
+          </p>
+
+          <div className="mb-2">
+            <Link isExternal href="https://electricitymaps.com/privacy-policy">
+              Privacy Policy
+            </Link>
+          </div>
+
+          <div className="mb-4">
+            <Link isExternal href="https://electricitymaps.com/legal-notice">
+              Legal Notice
+            </Link>
+          </div>
+
+          <p className="pb-2 text-xs text-secondary dark:text-secondary-dark">
+            App version: {APP_VERSION}
```

Comment:
We have a translated string for this we can use instead. that way it will match the language.

Reply:
Will add!